sklearn/metrics/_plot/det_curve.py

```diff
@@ -431,14 +436,15 @@ def plot_det_curve(
     Examples
     --------
     >>> import matplotlib.pyplot as plt
-    >>> from sklearn import datasets, metrics, model_selection, svm
-    >>> X, y = datasets.make_classification(random_state=0)
-    >>> X_train, X_test, y_train, y_test = model_selection.train_test_split(
-    ...     X, y, random_state=0)
-    >>> clf = svm.SVC(random_state=0)
-    >>> clf.fit(X_train, y_train)
-    SVC(random_state=0)
-    >>> metrics.plot_det_curve(clf, X_test, y_test) # doctest: +SKIP
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.metrics import plot_det_curve
+    >>> from sklearn.model_selection import train_test_split
+    >>> from sklearn.svm import SVC
+    >>> X, y = make_classification(n_samples=1000, random_state=0)
+    >>> X_train, X_test, y_train, y_test = train_test_split(
+    ...     X, y, test_size=0.4, random_state=0)
+    >>> clf = SVC(random_state=0).fit(X_train, y_train)
+    >>> plot_det_curve(clf, X_test, y_test)
```

Comment:
Here we will raise a deprecation warning. It might be best to keep the skip then.